programs/manifest/src/state/market.rs

```diff
@@ -801,6 +803,14 @@ impl<Fixed: DerefOrBorrowMut<MarketFixed>, Dynamic: DerefOrBorrowMut<[u8]>>
         }
         insert_order_into_tree(is_bid, fixed, dynamic, free_address, &resting_order);
 
+        get_mut_helper::<RBNode<RestingOrder>>(dynamic, free_address)
+            .set_payload_type(MarketDataTreeNodeType::RestingOrder as u8);
+        solana_program::msg!("added at {:?}", free_address);
+        solana_program::msg!(
+            "{}",
+            get_helper::<RBNode<RestingOrder>>(dynamic, free_address).get_payload_type() as u8
+        );
```

Comment:
should use trace macro

Reply:
gonna get rid of that debug logging actually